compiler/rustc_feature/src/active.rs

```diff
@@ -522,6 +522,8 @@ declare_features! (
     /// Allows creation of instances of a struct by moving fields that have
     /// not changed from prior instances of the same struct (RFC #2528)
     (active, type_changing_struct_update, "1.58.0", Some(86555), None),
+    /// Enables rustc to generate code that instructs libstd to NOT ignore SIGPIPE.
+    (active, unix_sigpipe, "1.63.0", Some(97889), None),
```

Comment:
1.64.0 should be now?

Reply:
Yes that must be the right move, thanks, fixed.

BTW, regarding https://github.com/rust-lang/rust/pull/97802#discussion_r893965842: lto knew to remove the code with a `bool`, but since I changed it to a `u8` it does not. I added an "Unresolved Question" about it to the [tracking issue](https://github.com/rust-lang/rust/issues/97889).